agcaballero/portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,94 @@
+/* General */
+
+body {
+  font-family: 'Helvetica Nueue', Helvetica, Arial, sans-serif;
+  background-color: #f4f1de;
```

Comment:
The style guide recommends sorting declarations in alphabetical order:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

Reply:
Should the overall sections (e.g. tag names, class names etc) also be sorted within their sections (e.g. Navigation bar, About Page)?

Reply:
I didn't see anything about that aspect of ordering in the style guide, so I'm assuming that's optional and up to you.